client/components/OperationList.js

```diff
@@ -165,6 +165,142 @@ function ComputeAttachmentLink(op) {
     return `operations/${op.id}/`+file;
 }
 
+
+
+class LabelComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            editMode: false            
+        };
+    }
+
+    dom() {
+        return this.refs.customlabel.getDOMNode();
+    }
+
+    switchToEditMode() {
+        this.setState({ editMode: true }, function() {
+            this.dom().focus();
+            //We set the cursor at the end
+            try {
+                this.dom().selectionStart = this.props.defaultValue().length;
```

Comment:
Would this work: `this.dom().selectionStart = this.dom().value.length;` ? In this case, I think you can remove the try/catch